KEYMAPPING.md

```diff
@@ -0,0 +1,84 @@
+# Star Trek Fleet Command - Community Patch
```

Comment:
This header doesn't need to be in this file :)

Reply:
There should always be a level 1 header for Markdown files.  I kept the same title for consistency.